templates/workflow/workflow.tpl

```diff
@@ -150,6 +157,13 @@
 								>
 									{translate key="publication.createVersion"}
 								</pkp-button>
+								<pkp-button
+									v-if="submission.status === getConstant('STATUS_PUBLISHED') && workingPublication.status !== getConstant('STATUS_PUBLISHED')"
```

Comment:
I think we can remove the first condition, `submission.status === getConstant('STATUS_PUBLISHED')`. This will mean that the preview button will appear here for the very first version. I know it will duplicate the button at the top of the page, but I think that's ok. It's good to have two places for them to see it.